test-web-library-build/gulpfile.js

```diff
@@ -2,7 +2,9 @@
 
 let build = require('@microsoft/web-library-build');
 
+build.sass.setConfig({ useCSSModules: true });
```

Comment:
Because when you have projects that assume .foo will exist, globally, that would break them. It defaults to local scope, which is super rigid and breaks many assumptions.